Common/cpp/ReanimatedRuntime/WorkletRuntime.h

```diff
@@ -65,10 +65,10 @@ class WorkletRuntime : public jsi::HostObject,
   std::vector<jsi::PropNameID> getPropertyNames(jsi::Runtime &rt) override;
 
  private:
+  std::recursive_mutex runtimeMutex_;
```

Comment:
I think this should be moved because it's used in `WorkletRuntime::WorkletRuntime` constructor as an argument of `makeRuntime` before its own `std::recursive_mutex` default constructor was called.
While it will probably work because we are only passing the address there, I think it's safer to have it initialized already.

Reply:
That's a good point, and you're right! Yes, it works because we are passing a pointer instead of the value of the field